### ISSUE TITLE
stuck at NBXplorer is synchronizing after updating

### ISSUE THREAD

**Issue by adspirit** [btcpayserver/btcpayserver-docker#708]
Good day,

I was running on v1.4x and everything was working fine. Then updated to latest version, noticed that syncing was working fine but (NBXplorer is synchronizing) is stuck.

I have prune setup "opt-save-storage-s" and below is my log:

`info: Configuration:  Data Directory: /datadir
info: Configuration:  Configuration File: /datadir/Main/settings.config
info: Configuration:  Network: Mainnet
info: Configuration:  Supported chains: BTC
info: Configuration:  DBCache: 50 MB
info: Configuration:  Network: Mainnet
info: Configuration:  Supported chains: BTC
info: Configuration:  DBCache: 50 MB
info: NBXplorer.DatabaseSetup: Postgres services activated
info: Explorer:       Trimming the event table if needed...
warn: NBXplorer.PostgresMigration: A legacy DBTrie database has been previously migrated to postgres and is still present. You can safely delete it if you do not expect using it in the future. To delete the old DBTrie database, start NBXplorer with --deleteaftermigration (or environment variable: NBXPLORER_DELETEAFTERMIGRATION=1)
info: NBXplorer.Indexer.BTC: TCP Connection succeed, handshaking...
info: NBXplorer.Indexer.BTC: Handshaked
info: NBXplorer.Indexer.BTC: Testing RPC connection to http://bitcoind:43782/
Hosting environment: Production
Content root path: /app/
Now listening on: http://0.0.0.0:32838
Application started. Press Ctrl+C to shut down.
info: NBXplorer.Indexer.BTC: RPC connection successful
info: NBXplorer.Indexer.BTC: Full node version detected: 230000
info: NBXplorer.Indexer.BTC: NBXplorer is correctly whitelisted by the node
info: NBXplorer.Events: BTC: Node state changed: NotStarted => NBXplorerSynching
`

tried rebooting server and followed this guide but with no luck. 

Please advise.

Thanks and Regards,

**Comment by NicolasDorier** [comment]
The error seems to be your bitcoin node having troubles.
Can you check bitcoind logs?

**Comment by adspirit** [comment]
CPU:4 vCore
RAM: 8GB
SSD: 160 GB

In attempt to fix it I deleted the whole data dir and now it is validating blocks from start, currently at 75% will confirm the status soon

**Comment by NicolasDorier** [comment]
@adspirit for this SSD size, you might get issue with disk space, are you using pruning?

If not,

```bash
BTCPAYGEN_ADDITIONAL_FRAGMENTS="opt-save-storage-s"
. btcpay-setup.sh -i
```

**Comment by adspirit** [comment]
Yes I use pruning (s). Before deleting dir I tried different pruning (xs, xxs), pruneheight changed but still was stuck. 

currently it's at 86%

**Comment by NicolasDorier** [comment]
86% seems to me high. If you used pruning it shouldn't go that high. `-s` for example, should be around 70 GB

**Comment by ndeet** [comment]
@NicolasDorier I think he means the sync progress is 86% and not the disk usage

**Comment by adspirit** [comment]
> 86% seems to me high. If you used pruning it shouldn't go that high. `-s` for example, should be around 70 GB

I was talking about validating blocks, it finished validating but again stuck at NBXplorer sync

BTC
NBXplorer headers height: 757495
The node is synchronized (Height: 757495)
NBXplorer is synchronizing... (Height: 691671)


```
{
  "chain": "main",
  "blocks": 757496,
  "headers": 757496,
  "bestblockhash": "00000000000000000002e08d27f5f3c8a1a7cf04386f64edadf2103605124811",
  "difficulty": 31360548173144.85,
  "time": 1665135775,
  "mediantime": 1665131899,
  "verificationprogress": 0.9999990955028843,
  "initialblockdownload": false,
  "chainwork": "00000000000000000000000000000000000000003697666fdd119afd697df678",
  "size_on_disk": 48763181486,
  "pruned": true,
  "pruneheight": 721296,
  "automatic_pruning": true,
  "prune_target_size": 52428800000,
  "warnings": ""
}
```

I'm not sure how to resolve this issue.

**Comment by NicolasDorier** [comment]
With so many block behind for NBXplorer, my guess of what is happening is that NBXplorer can't request blocks that far because those have been pruned.
